File: app/src/main/java/com/mobdeve/s20/group7/mco2/StoreItem.java
package com.mobdeve.s20.group7.mco2;

public class StoreItem {
    private final String name;
    private final int pointsCost;
    private final int iconResource;

    public StoreItem(String name, int pointsCost, int iconResource) {
        this.name = name;
        this.pointsCost = pointsCost;
        this.iconResource = iconResource;
    }

    public String getName() {
        return name;
    }

    public int getPointsCost() {
        return pointsCost;
    }

    public int getIconResource() {
        return iconResource;
    }
}

File: app/src/main/java/com/mobdeve/s20/group7/mco2/models/StoreItem.kt
package com.mobdeve.s20.group7.mco2.models

data class StoreItem(
    val id: String = "",
    val name: String = "",
    val description: String = "",
    val price: Int = 0,
    val iconResId: Int = 0,
    var isPurchased: Boolean = false
)